codegenerator/generatemodel/integration_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/taskcluster/slugid-go/slugid"
+	"github.com/taskcluster/taskcluster-base-go/jsontest"
```

Comment:
I had some local utility methods for comparing raw json elements, and now i'm using them in a different package, so I moved them into their own utility package that I could use everywhere, hence the new import.